### DISCUSSION TITLE
Installing on Kubernetes

### DISCUSSION THREAD

**Discussion by JunseoChoJJ** [apache/superset#27774] [Q&A / Help]
Hello,

I am trying to install superset in kubernetes in local using minikube.

I followed the instructions here https://superset.apache.org/docs/installation/running-on-kubernetes
helm upgrade --install --values my-values.yaml superset superset/superset
when I try this, 
kubectl get pods
NAME                                    READY   STATUS      RESTARTS   AGE
superset-celerybeat-7cdcc9575f-k6xmc    1/1     Running     0          119s
superset-f5c9c667-dw9lp                 1/1     Running     0          4m7s
superset-f5c9c667-fk8bk                 1/1     Running     0          4m11s
superset-init-db-zlm9z                  0/1     Completed   0          111s
superset-postgresql-0                   1/1     Running     0          6d20h
superset-redis-master-0                 1/1     Running     0          6d20h
superset-worker-75b48bbcc-jmmjr         1/1     Running     0          4m8s
superset-worker-75b48bbcc-qrq49         1/1     Running     0          4m12s

It should be look like this, However my status is crash or pending
I think yaml file is the problem

Could you explain me how to write yaml file in certain way. I used default values yaml and it doesn't works

**Comment by littletuna4** [comment]
I'm in the same boat.
For me at least, Inspecting the logs of the crashed pods indicated that the service refuses to start if you have default passwords.

With that said, I've just starting with k8's/superset and I've still not worked out how to get it started.

**Reply by JunseoChoJJ** [reply]
I figure out 
my problem was I need to give a pvc in postgresql 

in your case,
configOverrides:
  secret: |
    SECRET_KEY = 'YOUR_OWN_RANDOM_GENERATED_SECRET_KEY'

you should change key part here 
https://superset.apache.org/docs/installation/kubernetes

you might get an erro on pvc pod problem as well. 
https://github.com/hashicorp/consul-helm/issues/237
see this.
